src/Compilers/CSharp/Test/Symbol/Symbols/CovariantReturnTests.cs

```diff
@@ -336,13 +593,13 @@ public class Derived : Base
     private void SuppressUnusedWarning() => E?.Invoke();
 }
 ";
-            var comp = CreateCompilation(source, parseOptions: TestOptions.WithoutCovariantReturns).VerifyDiagnostics(
+            var comp = CreateCompilationWithoutCovariantReturns(source).VerifyDiagnostics(
```

Comment:
is there a rule that event overrides must have invariant returns?

Reply:
Yes, that is the existing language restriction, and nothing about this feature relaxes it. It has to be that way, as the event type is a parameter type of its accessors.

---
In reply to: [421069069](https://github.com/dotnet/roslyn/pull/43797#discussion_r421069069) [](ancestors = 421069069)